pkg/models/saml.go

```diff
@@ -0,0 +1,5 @@
+package models
+
+type IsSAMLEnabledCommand struct {
```

Comment:
Solely for my understanding, I'm assuming this will eventually implement whatever is needed to fetch `Enabled` from the configuration, right?

Reply:
To get this working we would implement a handler in enterprise that handled structs of this type that set the Result to `true` or `false` depending on the setting.

Torkel suggested we skip this though and have the enabled flag live in oss. So I will remove this and add the enabled flag to oss instead.